types/wordpress__customize/Panel_Constructor.d.ts

```diff
@@ -0,0 +1,5 @@
+import { ThemesPanel } from './ThemesPanel';
+
+export interface Panel_Constructor {
+	themes: ThemesPanel;
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/wordpress__customize/Panel_Constructor.d.ts;
  Line: 4;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.